pkg/kubelet/cm/dra/plugin/plugin.go

```diff
@@ -59,41 +58,48 @@ func (h *RegistrationHandler) RegisterPlugin(pluginName string, endpoint string,
 	return nil
 }
 
-// Return the highest supported version
+// Return the highest supported version.
 func highestSupportedVersion(versions []string) (*utilversion.Version, error) {
 	if len(versions) == 0 {
 		return nil, errors.New(log("DRA driver reporting empty array for supported versions"))
 	}
 
 	var highestSupportedVersion *utilversion.Version
+
 	var theErr error
+
 	for i := len(versions) - 1; i >= 0; i-- {
 		currentHighestVer, err := utilversion.ParseGeneric(versions[i])
 		if err != nil {
 			theErr = err
+
 			continue
 		}
+
 		if currentHighestVer.Major() > 1 {
 			// CSI currently only has version 0.x and 1.x (see https://github.com/container-storage-interface/spec/releases).
 			// Therefore any driver claiming version 2.x+ is ignored as an unsupported versions.
-			// Future 1.x versions of CSI are supposed to be backwards compatible so this version of Kubernetes will work with any 1.x driver
-			// (or 0.x), but it may not work with 2.x drivers (because 2.x does not have to be backwards compatible with 1.x).
+			// Future 1.x versions of CSI are supposed to be backwards compatible so this version of Kubernetes
+			// will work with any 1.x driver (or 0.x), but it may not work with 2.x drivers
+			// (because 2.x does not have to be backwards compatible with 1.x).
```

Comment:
CSI?

Reply:
thanks. I left only this comment: // DRA currently only has version 0.x